packages/server/src/fhir/operations/getwsbindingtoken.ts

```diff
@@ -40,8 +41,16 @@ export async function getWsBindingTokenHandler(req: Request, res: Response): Pro
   }
 
   const subscriptionId = req.params.id;
-  const subExists = await redis.exists(`Subscription/${subscriptionId}`);
-  if (!subExists) {
+  const cacheEntryStr = await redis.get(`Subscription/${subscriptionId}`);
```

Comment:
Hm, we probably should not be reaching into Redis directly here.  We should go through `repo` to make sure that it's respecting user and project permissions.

Reply:
i.e.,

<img width="739" alt="image" src="https://github.com/medplum/medplum/assets/749094/832db7c0-9779-4e6b-b606-54a1e51bfe21">

This also protects against implementation detail changes (how resources are cached, whether caching is enabled, etc).

Reply:
Yeah good call. 

I don't think a user should be able to get here without being able to access this subscription resource anyways but it's probably better for a number of reasons